### PR TITLE
Fix #25

### LINKED ISSUE #25: GLPK windows binary unavailable
```
$ wget http://downloads.sourceforge.net/project/winglpk/winglpk/GLPK-4.52/winglpk-4.52.zip
--2015-07-24 19:43:42--  http://downloads.sourceforge.net/project/winglpk/winglpk/GLPK-4.52/winglpk-4.52.zip
Resolving downloads.sourceforge.net (downloads.sourceforge.net)... 216.34.181.59
Connecting to downloads.sourceforge.net (downloads.sourceforge.net)|216.34.181.59|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-07-24 19:43:45 ERROR 404: Not Found.
```
Is this a temporary thing?
CC @tkelman

### PR DESCRIPTION
**Pull request by tkelman**
fixes #25

### CODE REVIEW

File: deps/build.jl [hunk 1]
```diff
@@ -56,35 +55,7 @@ includedirs = String["$(julia_usrdir)/include"]
...
-            ChangeDirectory(glpkdlldirw(64))
-            FileRule(glpkdestw(64), @build_steps begin
-                `cp $(glpkdllname).dll $(glpkdestw(64))`
-            end)
-        end
-    end), glpkdep, os = :Windows)
-
-@windows_only push!(BinDeps.defaults, BuildProcess)
+provides(Binaries, URI("https://bintray.com/artifact/download/tkelman/generic/win$glpkname.zip"),
+         glpkdep, unpacked_dir="$glpkname/w$WORD_SIZE", os = :Windows)
```
Comment (mlubin) [inline deps/build.jl]: Did you repack the dlls?

Comment (tkelman) [inline deps/build.jl]: Nope, didn't touch it - I implemented this `unpacked_dir` feature for binaries more recently than this code was last touched

Comment (mlubin) [inline deps/build.jl]: Oh, cool

**Comment by mlubin** [comment]
@carlobaldassi might appreciate some squashing

**Comment by mlubin** [comment]
Given the urgency of glpk being currently broken on windows, I'll merge and tag once tests pass.

**Pull request merged**